### ISSUE TITLE
[WORK NOT APPROVED] Implement updated intro animatic [WIP]

### ISSUE THREAD

**Issue by IMAFIREHAZARD** [smclaughlan/GWJ52#201]
![image](https://user-images.githubusercontent.com/101430646/208220558-092e2565-fb13-480e-b5e4-1c1e67a632b6.png)
Storyboard of new intro animatic

Still waiting on final art and dialog assets

**Comment by IMAFIREHAZARD** [comment]
![image](https://user-images.githubusercontent.com/101430646/208220993-276426f7-197e-4998-a1a8-6fb67a0e1b75.png)

**Comment by IMAFIREHAZARD** [comment]
Work not approved for this item. Closing as won't fix.
